packages/http-client-python/eng/scripts/Build-Packages.ps1

```diff
@@ -47,19 +47,21 @@ New-Item -ItemType Directory -Force -Path "$outputPath/packages" | Out-Null
 Write-Host "Getting existing version"
 $emitterVersion = node -p -e "require('$packageRoot/package.json').version"
 
-# build the generator jar
+# build the generator
 Push-Location "$packageRoot/generator"
 
-# build and pack the emitter with the generator jar
+# build and pack the emitter with the generator
 Push-Location "$packageRoot"
 try {
     Write-Host "Working in $PWD"
 
     Invoke-LoggedCommand "npm run build" -GroupOutput
 
-    # after python 3.13 fix pylint issue, will reopen the check
-    # Write-Host "run lint check for pygen"
-    # Invoke-LoggedCommand "npm run lint:py" -GroupOutput
+    # Only run lint:py on Linux OS
+    if ($IsLinux) {
+        Write-Host "run lint check for pygen"
+        Invoke-LoggedCommand "npm run lint:py" -GroupOutput
+    }
```

Comment:
Windows CI report the following error which is not reported in Linux and I can't reproduce it in my local windows. So it is environment issue instead of code issue. As long as the lint check in Linux works, we could make sure there is no lint issues in py source code.
![image](https://github.com/user-attachments/assets/32d8fe02-b8bb-4f66-abae-661d33d3b748)